lib/internal/util.js

```diff
@@ -865,7 +866,13 @@ function getCIDR(address, netmask, family) {
 }
 
 const handleTypes = ['TCP', 'TTY', 'UDP', 'FILE', 'PIPE', 'UNKNOWN'];
+setOwnProperty(handleTypes, -1, 'INVALID');
+
 function guessHandleType(fd) {
+  if (!NumberIsInteger(fd)) {
```

Comment:
In fs APIs we will throw an error if the `fd` is the incorrect type. We probably should do so here for consistency